library/Zend/InputFilter/InputFilterAbstractServiceFactory.php

```diff
@@ -52,9 +49,8 @@ public function canCreateServiceWithName(ServiceLocatorInterface $inputFilters,
      * @param string                  $rName
      * @return \Zend\InputFilter\InputFilterInterface
      */
-    public function createServiceWithName(ServiceLocatorInterface $inputFilters, $cName, $rName)
```

Comment:
No, it can stay just the way it was. Regarding the method `getServiceLocator` there already is an other issue for this and I think it depends on the outcome of that discussion whether it needs a change to ` AbstractPluginManager`. See #7388.